doc/source/enhancements.rst

```diff
@@ -8,11 +8,12 @@ Built-in enhancement methods
 stretch
 -------
 
-The most basic operation is to stretch the image so that the data fits
-to the output format.  There are many different ways to stretch the
-data, which are configured by giving them in `kwargs` dictionary, like
-in the example above.  The default, if nothing else is defined, is to
-apply a linear stretch.  For more details, see below.
+The most basic operation is to stretch the image so that the data fits to
+the output format.  There are many different ways to stretch the data,
+which are configured by giving them in `kwargs` dictionary, like in the
+example above.  The default, if nothing else is defined, is to apply
+a linear stretch.  For more details, see
+:ref:`enhancing the images <enhancing-the-images>`.
```

Comment:
Have you tried rendering the HTML locally? Does this link show up properly? Just want to make sure before merging.

Reply:
HTML output after `make html`:

```html
<p>The most basic operation is to stretch the image so that the data fits to
the output format.  There are many different ways to stretch the data,
which are configured by giving them in <cite>kwargs</cite> dictionary, like in the
example above.  The default, if nothing else is defined, is to apply
a linear stretch.  For more details, see
<a class="reference internal" href="composites.html#enhancing-the-images"><span class="std std-ref">enhancing the images</span></a>.</p>
```

I checked it interactively and the link shows up correctly.